File: common/src/main/java/com/ssblur/scriptor/effect/EmpoweredStatusEffect.java
package com.ssblur.scriptor.effect;

import net.minecraft.world.effect.MobEffect;
import net.minecraft.world.effect.MobEffectCategory;
import net.minecraft.world.entity.LivingEntity;

public class EmpoweredStatusEffect extends MobEffect {
  float scale;
  public EmpoweredStatusEffect(float scale) {
    super(MobEffectCategory.BENEFICIAL, 0x565656);
    this.scale = scale;
  }

  @Override
  public boolean applyEffectTick(LivingEntity entity, int amplifier) {
    return true;
  }

  public float getScale() {
    return scale;
  }
}

File: common/src/main/kotlin/com/ssblur/scriptor/effect/EmpoweredStatusEffect.kt
package com.ssblur.scriptor.effect

import net.minecraft.world.effect.MobEffect
import net.minecraft.world.effect.MobEffectCategory
import net.minecraft.world.entity.LivingEntity

class EmpoweredStatusEffect(var scale: Float) :
    MobEffect(MobEffectCategory.BENEFICIAL, 0x565656) {
    override fun applyEffectTick(entity: LivingEntity, amplifier: Int): Boolean {
        return true
    }
}